src/main/java/frc/robot/Robot.java

```diff
@@ -27,19 +28,27 @@
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
```

Comment:
Rename m_chooser to autonChooser to reduce potential confusion

Reply:
done